src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,59 @@
+package calculator;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("문자열 계산기 단위 테스트")
+public class StringCalculatorTest {
+
+    StringCalculator calculator;
```

Comment:
테스트 클래스의 인스턴스도 외부에서 접근할 수 없도록 `private`하게 제한해주는게 좋을 것 같네요. 😁
```suggestion
    private StringCalculator calculator;
```